Cubical/ZCohomology/Properties.agda

```diff
@@ -195,31 +195,23 @@ rightInv (Iso-coHom-coHomRed {A = A , a} n) =
                       (Iso.fun (PathIdTruncIso (suc n)) (isContr→isProp (isConnectedKn n) ∣ f a ∣ ∣ 0ₖ _ ∣))
 leftInv (Iso-coHom-coHomRed {A = A , a} n) =
   sElim (λ _ → isOfHLevelPath 2 § _ _)
-        λ {(f , p) → cong ∣_∣₂ (ΣPathP (((funExt λ x → cong (λ y → f x -ₖ y) p
+        λ {(f , p) → cong ∣_∣₂ (ΣPathP (((funExt λ x → (cong (λ y → f x -ₖ y) p
                                                     ∙∙ cong (λ y → f x +ₖ y) -0ₖ
-                                                    ∙∙ rUnitₖ _ (f x)))
+                                                    ∙∙ rUnitₖ _ (f x)) ∙ refl))
```

Comment:
Why compose with refl?

Reply:
This is due to a change in rCancelₖ (with the new definition of subtraction). Adding refl here makes the path algebra in the helper function below easier.